sdk/communication/Azure.Communication.Administration/src/PhoneNumberSearchCancelOperation.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Communication.Administration.Models;
+
+namespace Azure.Communication.Administration
+{
+    /// <summary>
+    /// Represent a long-running phone number search cancellation operation.
+    /// </summary>
+    public class PhoneNumberSearchCancelOperation : Operation<PhoneNumberSearch>
```

Comment:
I thought we decided to remove all the operations and bring them back after the redesign